rust/connlib/tunnel/Cargo.toml

```diff
@@ -25,6 +25,7 @@ domain = "0.8"
 boringtun = { workspace = true }
 chrono = { workspace = true }
 pnet_packet = { version = "0.34" }
+futures-bounded = { git = "https://github.com/libp2p/rust-libp2p", branch = "feat/stream-map" }
```

Comment:
@conectado Are you okay with a git-dependency here for now or would you like to wait for a release?

I've proposed the changes in https://github.com/libp2p/rust-libp2p/pull/4616 but it might take a few days before this is landed.

Reply:
I don't mind the git-dependency